File: Bhumika_Capstone/DataBase_DB/src/main/java/com/travelbooking/logger/Logger.java
package com.travelbooking.logger;

import org.slf4j.LoggerFactory;

public final class Logger {

    private static final org.slf4j.Logger LOGGER = LoggerFactory.getLogger(Logger.class);

    private Logger() {
    }

    public static void info(String message) {
        LOGGER.info(message);
    }

    public static void warn(String message) {
        LOGGER.warn(message);
    }

    public static void error(String message) {
        LOGGER.error(message);
    }

    public static void debug(String message) {
        LOGGER.debug(message);
    }
}

File: UI/logger/Logger.ts
export class Logger {

    static info(message: string) {

        console.log(`[INFO] ${message}`);

    }

    static success(message: string) {

        console.log(`[PASS] ${message}`);

    }

    static error(message: string) {

        console.log(`[FAIL] ${message}`);

    }

}